docker-compose update - Closes #786 (#792)
LiskHQ/lisk-service: README.md

The default configuration is sufficient to run Lisk Service against the local node.

Before running the application copy the default docker-compose environment file:

```bash
cp docker/example.env .env
```

In the next step, set the required environment variables.

```bash

> nano .env # or vim

Reviewer: ```suggestion
$EDITOR .env
```

```

> The example below is valid with the assumption the Lisk Core node is running on host machine, not in a Docker container.

Reviewer 2: ```suggestion
The example below is valid with the assumption the Lisk Core node is running on the host machine, not inside of a Docker container.
```

Reviewer: "The example assumes that [...]"?

```
## Required
# The local Lisk Core node WebSocket API port
export LISK_CORE_WS="ws://host.docker.internal:8080"
```

> When running a Docker instance of Lisk Core that is run by Docker, the variable needs to refer to the container: `LISK_CORE_WS="ws://<your_docker_container>:8080"`.

Reviewer 2: ```suggestion
When running Lisk Core inside of a Docker container, the variable needs to refer to the container: `LISK_CORE_WS="ws://<your_docker_container>:8080"`.
```

Reviewer: And both containers need to have a network in common.

make down
```

> Optional: It is possible to use regular docker-compose commands such as `docker-compose up -d`. Please check the `Makefile` for more examples.

## Benchmark

> Assuming lisk-service is running on localhost:9901, and you are in the root of this repo, you can run:

Reviewer 3: ```suggestion
Assuming lisk-service is running on the localhost:9901, and you are in the root of this repo, you can run the following:
```
